src/main/java/org/spongepowered/api/statistic/achievement/Achievement.java

```diff
@@ -75,89 +60,17 @@ static Builder builder() {
-     * If this achievement is backed by a statistic (eg. if
-     * {@link #getSourceStatistic()} is not absent) then this will return the
-     * target value which must be reached for this achievement to be awarded to
-     * a player.
+     * Returns true if this is a "special" achievement.
      *
-     * @return The target value, if available
-     */
-    Optional<Long> getStatisticTargetValue();
-
-    /**
-     * Represents a builder interface to create new and custom instances of
-     * {@link Achievement}s.
+     * @return True if special achievement
      */
-    interface Builder extends ResettableBuilder<Achievement, Builder> {
-
-        /**
-         * Sets the internal name for the {@link Achievement}.
-         *
-         * @param name The name of this achievement
-         * @return This builder, for chaining
-         */
-        Builder name(String name);
-
-        /**
-         * Sets the translation for the {@link Achievement}.
-         *
-         * @param translation The translation for the achievement
-         * @return This builder, for chaining
-         */
-        Builder translation(Translation translation);
-
-        /**
-         * Sets the description that describes this {@link Achievement}.
-         *
-         * @param description The description of this achievement
-         * @return This builder, for chaining
-         */
-        Builder description(Translation description);
-
-        /**
-         * Sets the parent of this {@link Achievement}, if there is one.
-         *
-         * @param parent The parent of this achievement
-         * @return This builder, for chaining
-         */
-        Builder parent(@Nullable Achievement parent);
-
-        /**
-         * Sets a statistic which will be used for tracking this achievement. May be
-         * null if this achievement is not backed by a statistic but is manually
-         * awarded instead. If the statistic is not null then the
-         * {@link #targetValue(long)} must be set as well. This defaults to null if
-         * not set.
-         *
-         * @param stat The statistic, or null if not backed by a statistic
-         * @return This builder, for chaining
-         */
-        Builder sourceStatistic(@Nullable Statistic stat);
-
-        /**
-         * Sets the target value of the statistic backing this achievement. If the
-         * source statistic is not set then this value will be ignored if set.
-         * Defaults to 1 if not set.
-         *
-         * @param value The target value
-         * @return This builder, for chaining
-         */
-        Builder targetValue(long value);
-
-        /**
-         * Builds and registers an instance of an {@link Achievement}.
-         *
-         * @return A new instance of a achievement
-         * @throws IllegalStateException If the achievement is not completed
-         */
-        Achievement buildAndRegister() throws IllegalStateException;
+    boolean isSpecial();
```

Comment:
What is a "special" achievement? Is this pertaining the color of the achievement?